src/components/screens/TeamScreen/Contributors.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable no-undef */
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Avatar, styles, Link } from '@storybook/design-system';
+
+const { typography } = styles;
+
+const Heading = styled.div`
+  font-size: ${typography.size.m2}px;
+  font-weight: ${typography.weight.extrabold};
+  line-height: 28px;
+`;
+
+const Text = styled.h1`
+  font-size: ${typography.size.s3}px;
+  line-height: 28px;
+  margin-top: 12px;
+`;
+
+const Wrapper = styled.div`
+  padding: 1rem;
+`;
+
+const CommunityAvatars = styled.div`
+  margin-top: 24px;
+  margin-left: -10px;
+  margin-right: -10px;
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const AvatarWrapper = styled(Avatar).attrs({ size: 'large' })`
+  min-width: 40px;
+  margin: 10px;
+`;
+
+const StyledLink = styled(Link)`
+  margin-top: 12px;
+`;
+
+const PureContributors = ({ contributors }) => (
+  <Wrapper>
+    <section>
+      <Heading>+{parseInt(contributors.length / 3, 10)} Contributors</Heading>
```

Comment:
I think I can just send contributors from contributors function instead of doing it here. 😅

Reply:
Yeah good callout. I just did a quick look at the GitHub rest API and it looks like you can do something like this:

`https://api.github.com/repos/storybookjs/frontpage/contributors?per_page=2`

So maybe you can update the `contributorsUrl` below to include that query param with the appropriate value and then you won't have to worry at all about having to chop up the array?